articles/time_to_say_goodby.md

```diff
@@ -0,0 +1,55 @@
+ 
+4. Look for contact with the team member and their Superior
+Sometimes team members will be lured to directly change to another project. Sometimes this is not the case. Anyway, as a project manager I recommend to discuss with the superiors by when the team member is leaving the current project and to which share the person will work in the new project or can take over new tasks outside the current project. This agreement should be taken between Superior, project manaaer(ers) and team member in order that all have the same view of the situation.
+ 
+5. No project without any learnings
+Invite right beforehand all team members to a lessons learned meeting. Ensure that the meeting will either be split in separate ones if there are too many people involved for a common retro or search for someone to support you doing the lessons learned workshop. The workshop may be organized as a “world café”. Good practice, from my experience, is to make the project stages visible by putting cards on the wall for each month and each event during the project duration. I also saw such a review in way of a collage or story / poem. As a source of information, you can use the monthly status reports or your project diary. You will be amazed of what happened (and you already forgot about it) and what you and the team achieved. One important point for me is, that at the end of the meeting every single participant has the possibility to write his own lessons learned and take these notes with him into the next project. Maybe you can prepare some cards beforehand and hand them out as template that they can fill out. You can also ask your team members either for feedback to your job or if they want you to provide them feedback about themselves. To be prepared for these activities it is helpful to start writing a project diary and note some keywords each week what happened, what was good, what was bad, what are my learnings and also with your private project diary what happened to all team members so that you can see their development through the time being.
+
+ 6. “Goodbye and thank you for the fish”
+Most often project manager forget to take some budget in account for celebration of the project success or have a small give away as “Thank you” to the project members. No matter if you have any budget or not, it would be a good habit to invite your team members in advance to a project closing meeting. Purpose of this meeting is to officially hand over all responsibilities to the operation team and relieve the project team. That is also a good possibility to reflect the past months and perform a project review. Summarize the findings from your lessons learned meeting and share the impressions of that. This is the moment to say thank you. Maybe by some personal words to each member, a card with some keywords what was special to the person during the project duration, photos, etc... There are many possibilities to show recognition without expensive gifts or events.
```

Comment:
Klarer: "it would be a good habit to invite yo" -> "it is a good habit to invite yo"

Sprache "ll responsibilities to the operation team" -> "ll responsibilities to the operations team"

Inhalt: "That is also a good possibility to reflect the past months and perform a project review. Summarize the findings from your lessons learned meeting and share the impressions of that. " -> Ist das nicht schon mit Punkt 5 abgedeckt? Weglassen?